website/docs/r/workflow.html.markdown

```diff
@@ -63,38 +39,86 @@ resource "newrelic_workflow" "foo" {
 
 The following arguments are supported:
 
-* `account_id` - (Optional) Determines the New Relic account where the workflow will be created. Defaults to the account associated with the API key used.
 * `name` - (Required) The name of the workflow.
-* `enrichments_enabled` - (Optional) Whether enrichments are enabled..
-* `destinations_enabled` - (Optional) Whether destinations are enabled..
-* `enabled` - (Optional) Whether workflow is enabled.
-* `muting_rules_handling` - (Required) Specifies if the workflow should send notifications for muted issues. An issue can be either "Fully muted", "Partially muted" or "Not muted" (for more on these states go to [this](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications/#workflow-behavior) page). One of:
-  * DONT_NOTIFY_FULLY_MUTED_ISSUES - Do not send notifications for fully muted issues.
-  * DONT_NOTIFY_FULLY_OR_PARTIALLY_MUTED_ISSUES - Do not send notifications for fully or partially muted issues.
-  * NOTIFY_ALL_ISSUES - Always send notifications.
-* `destination` - (Required) A nested block that contains a channel id.
-* `issues_filter` - (Required) The issues filter.  See [Nested issues_filter blocks](#nested-issues_filter-blocks) below for details.
-* `enrichments` - (Optional) A nested block that describes a workflow's enrichments. See [Nested enrichments blocks](#nested-enrichments-blocks) below for details.
+* `issues_filter` - (Required) A filter used to identify issues handled by this workflow. See [Nested issues_filter blocks](#nested-issues_filter-blocks) below for details.
+* `muting_rules_handling` - (Required) How to handle partially muted issues. See [Muting Rules](#muting-rules) below for details.
+* `destination` - (Required) Notification configuration. See [Nested destination blocks](#nested-destination-blocks) below for details.
+* `account_id` - (Optional) Determines the New Relic account in which the workflow is created. Defaults to the account defined in the provider section.
+* `enrichments_enabled` - (Optional) Whether enrichments are enabled. Defaults to true.
+* `destinations_enabled` - (Optional) **DEPRECATED** Whether destinations are enabled. Please use `enabled` instead:
+these two are different flags, but they are functionally identical. Defaults to true.
+* `enabled` - (Optional) Whether workflow is enabled. Defaults to true.
+* `enrichments` - (Optional) Workflow's enrichments. See [Nested enrichments blocks](#nested-enrichments-blocks) below for details.
 
 ### Nested `issues_filter` blocks
 
-Each workflow type supports a set of arguments for the `issues_filter` block:
+Issue filter defines how to identify issues that should be handled by the workflow. 
+It consists of a number of predicates. 
```

Comment:
nit: It consists of one or more predicates.

Reply:
done